stl/inc/format

```diff
@@ -2138,11 +2137,11 @@ _NODISCARD _OutputIt _Write_integral(
     }
 
     // long long -1 representation in binary is 64 bits + sign
-    array<char, 65> _Buffer;
-    const auto [_End, _Ec] = _STD to_chars(_Buffer.data(), _Buffer.data() + _Buffer.size(), _Value, _Base);
+    char _Buffer[65];

```

Comment:
I am still a tiny bit unhappy about that `65`

No change requested though

Reply:
I mean it appears in one place, so I think just writing it there is best. That way the reader knows that it's only one place, and doesn't need to grep for the constant to find out.